index.html

```diff
@@ -6,7 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="styles.css">
   <script src="data.js"></script>
-  <script src="index.js"></script>
+  <script src="index.js" defer ></script>
 
   <title>Travel</title>
 </head>
```

Comment:
Nice job catching why the two files were not on speaking terms and fixing it with defer.